src/m365/aad/commands/m365group/m365group-remove.ts

```diff
@@ -115,6 +109,124 @@ class AadM365GroupRemoveCommand extends GraphCommand {
       }
     }
   }
+
+  private async getM365GroupSiteURL(logger: Logger, id: string): Promise<string> {
```

Comment:
As we're spelling URL as `Url` in other variables, let's align this name consistently as `getM365GroupSiteUrl`